### ISSUE TITLE
Android app not syncing anymore

### ISSUE THREAD

**Issue by letscho123** [etesync/android#172]
Hello,

I have been using etesync for more than 2 months now without problems. However, I noticed that the app is not syncing anymore since 5 days. Neither with Tasks.org nor with the calender. 
I tried to sync the app manually and it says "synchronizing now" for like 2 seconds, but nothing is happening. So I reinstalled it, logged in, but the problem persists. Now all Contacts, Calender etc. are empty. There is also no error message or anything and I can log in normally. Also my account information in the app is accessible.
The webversion seems to work fine. If I create an event there, I can see it in Thunderbird. But the latest changelogs from my phone are 5 days old.

I did not do any major changes on my phone, which could cause this issue and I did not use the etesync app actively for quite some time.

Android 11 CalyxOS (with latest updates)
Etesync 2.2.4
Tasks.org 11.7 (same issue with version 11.6)

Any help is very much appreciated!

**Comment by abaker** [comment]
Mentioning https://github.com/tasks/tasks/issues/1411 in case they're related

**Comment by letscho123** [comment]
Yeah I had the same issue. However, by removing my etesync account from tasks and putting it in again, the problem was solved there. So the syncing works again as it should on tasks.org, but the etesync app itself is still not syncing.

**Comment by abaker** [comment]
Did you enter your credentials directly into Tasks, or are you using the EteSync Android client to sync with Tasks?

**Comment by letscho123** [comment]
I entered them now directly in Tasks. I don't remember how I set it up the first time.

**Comment by letscho123** [comment]
The problem resolved itself by restarting my phone. Still strange that it happened in the first place.
